### PR TITLE
Remove plaintext secret option

### PR DESCRIPTION
**Pull request by kgcarr**
This is the first part to address issue #67 

We will not allow plaintext in the CR for username and password...so we will only allow random string admin and password 

We will have to target adding a user supplied secret in Q2

### CODE REVIEW

**Comment by horis233** [comment]
@kgcarr Please update the API as well
https://github.com/IBM/ibm-mongodb-operator/blob/master/pkg/apis/operator/v1alpha1/mongodb_types.go#L34

File: pkg/controller/mongodb/mongodb_controller.go [hunk 1]
```diff
@@ -160,19 +160,11 @@ func (r *ReconcileMongoDB) Reconcile(request reconcile.Request) (reconcile.Resul
...
-	}
-
-	if instance.Spec.MongoDBUser == "" {
-		user = createRandomAlphaNumeric(8)
-	} else {
-		user = instance.Spec.MongoDBUser
-	}
-
+	user = createRandomAlphaNumeric(8)
+	pass = createRandomAlphaNumeric(13)
```
Comment (chenzhiwei) [inline pkg/controller/mongodb/mongodb_controller.go]: how to make this consist during the mongodb operator restart?

Comment (kgcarr) [reply pkg/controller/mongodb/mongodb_controller.go]: @chenzhiwei the deployment supports changing the admin password on restarts, so it's ok for the admin secret to change

**Pull request merged**